apps/examples/wc-next13/next.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
```

Comment:
I think we shouldn't use strict mode and stay consistent with the creat-react-app example as strict mode 

here are more details https://github.com/justeattakeaway/pie/pull/309#discussion_r1136929579